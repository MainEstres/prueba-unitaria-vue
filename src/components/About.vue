<script setup>


</script>

<template>

    <div>
        <h1>
            Esto es una pagina de About
        </h1>
    </div>


</template>