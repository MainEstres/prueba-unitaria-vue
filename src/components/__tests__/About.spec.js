import { shallowMount } from "@vue/test-utils";
import { describe, expect, test } from "vitest";
import About from "../About.vue";

describe('Componente About', ()=>{
  test('Comparando snapshot', ()=>{

    const wrapper = shallowMount(About);

    expect(wrapper.html()).toMatchSnapshot();


  })
})