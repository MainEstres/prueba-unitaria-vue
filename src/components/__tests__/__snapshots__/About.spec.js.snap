// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Componente About > Comparando snapshot 1`] = `
"<div>
  <h1> Esto es una pagina de About </h1>
</div>"
`;
